make.php

```diff
@@ -111,92 +132,22 @@
     }
 }
 
-$imageNamesExtended = array_merge(
-    ...array_map(function ($targetName) use ($imageNames) {
-        return array_map(function($imageName) use ($targetName) {
-            return $imageName . ($targetName === 'base' ? '' : '-' . $targetName);
-        }, $imageNames);
-    }, $targetNames)
-);
-
-$codefreshFile = 'version: "1.0"
-stages:
-  - prepare
-' . implode("\n", array_map(function ($targetName) use ($cfLabelFromName) {
-    return '  - ' . $cfLabelFromName('build_', $targetName);
-}, $targetNames)) . '
-  - test
-  - push
-steps:
-  main_clone:
-    stage: prepare
-    type: git-clone
-    repo: atk4/image
-    revision: "${{CF_BRANCH}}"
-
-' . implode("\n\n", array_map(function ($targetName) use ($imageNames, $cfLabelFromName) {
-    return '  ' . $cfLabelFromName('build_', $targetName) . ':
-    type: parallel
-    stage: ' . $cfLabelFromName('build_', $targetName) . '
-    steps:
-' . implode("\n", array_map(function ($imageName) use ($targetName, $cfLabelFromName) {
-        return '      ' . $cfLabelFromName('b', $imageName . ($targetName === 'base' ? '' : '-' . $targetName)) . ':
-        type: build
-        image_name: atk4/image
-        target: ' . $targetName . '
-        tag: "${{CF_BUILD_ID}}-' . $imageName . ($targetName === 'base' ? '' : '-' . $targetName) . '"
-        registry: atk4
-        dockerfile: data/' . $imageName . '/Dockerfile';
-    }, $imageNames));
-}, $targetNames)) . '
-
-  test:
-    type: parallel
-    stage: test
-    steps:
-' . implode("\n", array_map(function ($imageName) use ($cfLabelFromName) {
-    return '      ' . $cfLabelFromName('t', $imageName) . ':
-        image: "atk4/image:${{CF_BUILD_ID}}-' . $imageName . '"
-        registry: atk4
-        commands:
-          - php test.php';
-}, $imageNamesExtended)) . '
-
-  push:
-    type: parallel
-    stage: push
-    when:
-      branch:
-        only:
-          - master
-    steps:
-' . implode("\n", array_map(function ($imageName) use ($genImageTags, $cfLabelFromName) {
-    return implode("\n", array_map(function ($imageTag) use ($cfLabelFromName, $imageName) {
-        return '      ' . $cfLabelFromName('p', $imageTag) . ':
-        candidate: "${{' . $cfLabelFromName('b', $imageName) . '}}"
-        type: push
-        registry: atk4
-        tag: "' . $imageTag . '"';
-    }, $genImageTags($imageName)));
-}, $imageNamesExtended)).'
-';
-file_put_contents(__DIR__ . '/.codefresh/deploy-build-image.yaml', $codefreshFile);
-
 
 $ciFile = 'name: CI
 
 on:
   pull_request:
   push:
   schedule:
-    - cron: \'20 */2 * * *\'
+    - cron: \'20 2 * * *\'
 
 jobs:
   unit:
     name: Templating
+    if: github.event_name != \'push\' || github.ref == \'refs/heads/master\' || github.ref == \'refs/heads/develop\'
     runs-on: ubuntu-latest
     container:
-      image: atk4/image
+      image: ghcr.io/mvorisek/image-php
```

Comment:
could we stick with atk4/ org owned repo please?

Reply:
done